website/docs/components/reveal/partials/code/how-to-use.md

```diff
@@ -0,0 +1,29 @@
+## How to use this component
+
+The `Reveal` component renders a button which triggers the display of additional content. The additional content can consist of either plain text or HTML and can include interactive elements such as links.
```

Comment:
Suggestion: 
```suggestion
The Reveal component renders a button which triggers the display of additional content. The additional content can consist of either plain text or HTML and can include interactive elements such as links.
```

Looking through other components and it doesn't seem like we put them in code quotes (but we also have inconsistencies, tracked with [HDS-1974](https://hashicorp.atlassian.net/browse/HDS-1974)).